portafly/src/App.tsx

```diff
@@ -0,0 +1,89 @@
+import 'react-app-polyfill/ie11'
+import { Brand } from '@patternfly/react-core'
+import React from 'react'
+import {BrowserRouter as Router, Redirect, useHistory} from 'react-router-dom'
+import { AppLayout, SwitchWith404, LazyRoute } from './components'
+import { LastLocationProvider } from 'react-router-last-location'
+import './App.css'
+import logo from './assets/logo.svg'
+
+const Logo = <Brand src={logo} alt={'patternfly logo'}/>
+const navItems = [
+  {
+    title: 'Overview',
+    to: '/',
+    exact: true,
+  },
+  {
+    title: 'Analytics',
+    to: '/analytics',
+    items: [
+      {to: '/analytics/usage', title: 'Usage'},
+    ],
+  },
+  {
+    title: 'Applications',
+    to: '/applications',
+    items: [
+      {to: '/applications', title: 'Listing'},
+      {to: '/applications/plans', title: 'Application Plans'}
+    ],
+  },
+  {
+    title: 'Integration',
+    to: '/integration',
+    items: [
+      {to: '/integration/configuration', title: 'Configuration'}
+    ]
+  }
+]
+
+const getOverviewPage = () => import('./pages/Overview')
+const getApplicationsPage = () => import('./pages/Applications')
+
+const App = () => {
+  return (
+      <Router>
+        <LastLocationProvider>
+          <Root />
+        </LastLocationProvider>
+      </Router>
+  )
+}
+
+const PagesSwitch = () => {
+  return (
+    <SwitchWith404>
+      <LazyRoute path="/" getComponent={getOverviewPage} />
+      <LazyRoute path="/applications" exact getComponent={getApplicationsPage} />
```

Comment:
Without `exact` routing is failing, always rendering Overview component
```suggestion
      <LazyRoute path="/" getComponent={getOverviewPage} exact />
      <LazyRoute path="/applications" exact getComponent={getApplicationsPage} exact />
```